pkg/test/executor/docker_entrypoint_test.go

```diff
@@ -301,7 +301,7 @@ type dockerfilePermutation struct {
 func createDockerfile(d dockerfilePermutation) string {
 	ep := "\nENTRYPOINT [\"/bin/echo\"]"
 	cmd := "\nCMD [\"echo\", \"This is from CMD\"]"
-	baseDockerFile := "FROM alpine:latest"
+	baseDockerFile := "FROM busybox:latest"
```

Comment:
@wdbaruni looks like this test is tightly coupled with Ubuntu distribution since they are failing in BuildKite